com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -72,9 +72,21 @@ static CoreEditorUtils()
         }
 
         // UI Helpers
+
+        /// <summary>Creates a 1x1 <see cref="Texture2D"/> with a plain <see cref="Color"/></summary>
+        /// <param name="color">The color to fill the texture</param>
+        /// <returns>a <see cref="Texture2D"/></returns>
+        public static Texture2D CreateColoredTexture2D(Color color)
+        {
+            Texture2D tex2 = new Texture2D(1, 1);
```

Comment:
I think it would be better to mark this texture as HideAndDontSave so its content survives a domain reload which is useful if you keep a reference of it in a window that can also survive a domain reload:
https://docs.unity3d.com/ScriptReference/HideFlags.HideAndDontSave.html

Reply:
Done thanks,